scripts/abi_artifact_generator/templates/types.h.jinja_template

```diff
@@ -0,0 +1,42 @@
+#pragma once
+/*( _ )*/
+
+/*#-
+This template file is distributed under the MIT License. See LICENSE.md for details.
+#*/
+
+/*(- generation_notice )*/
+
+#ifdef __SIZEOF_INT128__
+#define INT128_IS_SCALAR 1
+#else
+#define INT128_IS_SCALAR 0
+#endif
+
+#define IS_SCALAR(TYPE, VALUE) \
+  _Generic((VALUE), \
+    /*{- for type in supported_primitives }*/
+           /*( type )*/: 1, \
```

Comment:
Why `1` instead of `true`?

Reply:
Because that's `c` and `true` involves including `stdbool.h` (or whatever it's called), which caused problems with msvc.

Reply:
OK, add a comment for future generations.

Reply:
Done.